app/src/main/java/ch/epfl/sdp/blindly/match/Profile.kt

```diff
@@ -2,6 +2,7 @@ package ch.epfl.sdp.blindly.match
 
 import android.os.Parcel
 import android.os.Parcelable
+import ch.epfl.sdp.blindly.user.User
```

Comment:
Why was this import needed without any change in the file?

Reply:
It was when I tried to implement a function in this class. I deleted it, so I need to delete this import too. Thanks for seeing it !

Reply:
No worries !